content/projects/gsoc/2019/application.md

```diff
@@ -0,0 +1,120 @@
+
+#### Long description (Markdown)
+
+[Jenkins](https://jenkins.io/), originally founded in 2006 as "Hudson", is one of the leading automation servers available. Using an extensible, plugin-based architecture developers have created hundreds of plugins to adapt Jenkins to a multitude of build, test, and deployment automation workloads. Jenkins core is open-source ([MIT License](http://www.opensource.org/licenses/mit-license.php))
+
+The project has about 400 active contributors working on Jenkins core, plugins, website, project infrastructure, localization activities, etc. In total we have around 2000 different components including plugins, libraries, and various utilities. The main languages in the project are Java, Groovy and JavaScript, but we also have components written in other languages.
+
+This year we invite students to join the Jenkins community and to work together on Jenkins plugins in order to improve Jenkins user experience and reliability.
+
+
+#### Proposal process:
+
+In order to apply to the organization, please follow the [guidelines posted on our website](https://jenkins.io/projects/gsoc/students/#student-application-process). 
+
+Before submitting please go through the page (including the expectations section) and through the [Student guide](https://google.github.io/gsocguides/student/). Participating in Google Summer of Code is a serious commitment requiring an almost full-time dedication over several months. If it may overlap with your study and other commitments, we recommend to think twice before applying.
+
+If you have any questions about the application process, please feel free to contact us via the “jenkinsci-gsoc-orgs@googlegroups.com” mailing list.
+
+#### Proposal tags (max - 10)
+
+* new feature
+* user experience
+* plugins
+* performance
+* plugin developer tools
+* electronic design automation
+
+#### Contents
+
+* Chat page URL: https://gitter.im/jenkinsci/gsoc-sig
+* Mailing list page URL: https://jenkins.io/projects/gsoc/#contacts 
+* General-purpose e-mail: jenkinsci-gsoc-all-public@googlegroups.com
+
+### Application
+
+#### Why does your org want to participate in Google Summer of Code?
+
+In our community we are interested in more contributors in both Jenkins core and more than 1,000 of existing plugins. We consider Google Summer of Code as an opportunity to find new contributors and students interested in software development automation (continuous integration and continuous delivery). For us it is also a great opportunity to get existing contributors more involved into the community work. We have previously participated in GSoC 2016 and got much experience from it, especially regarding the student selection process. We also learned our lesson in 2017, we were able to attract more mentors interested in the project. We hope this will help us to improve student/mentor experience and overall results this year.
```

Comment:
Not sure I know what this means "We also learned our lesson in 2017..." I did not participate with Jenkins that year. Is it still applicable for the 2019 application? Maybe we can say something about SIGs and a more targeted mentor recruiting process? WDYT?

Reply:
Yes, it was a case in 2018. I think that we can just remove it from the draft

Reply:
```suggestion
In our community we are interested in more contributors in both Jenkins core and more than 1,000 of existing plugins. We consider Google Summer of Code as an opportunity to find new contributors and students interested in software development automation (continuous integration and continuous delivery). For us it is also a great opportunity to get existing contributors more involved into the community work. We also conducted [public retrospective after GSoC 2018](https://docs.google.com/document/d/1sJ9KIYHUoFWWE9HmoZC7HPDxm2i3uMFGQ2KKZ_1TTjk/edit?usp=sharing) and addressed feedback from students and mentors in our processes and documentation. We hope this will help us to improve student/mentor experience and overall results this year.
```